2017-11/README.md

```diff
@@ -0,0 +1,4 @@
+# Concurrency in Go: an introduction to channels.
+
+Channels are one of the most interesting features in GoLang. This talk shows some of the basics of working with concurrent applications, go routines and channels. This repository contains the demonstration code. The code is ocasionally intentionally broken to highlight different aspects of concurrency mechanisms. 
```

Comment:
Go~~Lang~~ Just Go. `#golang` is just used for a tag and the golang.org website (Disney owns go.com)

ocasionally -> occasionally (spell checkers are handy)

Reply:
Thanks for that! I use vim for my editor, and when coding all the red of the spell checker gets distracting, so its usually off.